x-pack/legacy/plugins/integrations_manager/common/types.ts

```diff
@@ -68,8 +65,8 @@ export interface AssetParts {
   type: AssetType;
   file: string;
 }
-
-export type AssetsGroupedByServiceByType = Record<ServiceName, Record<AssetType, AssetParts[]>>;
+export type Assets = Record<AssetType, AssetParts[]>;
```

Comment:
I read `Assets` as an Array not a Map/Record. Maybe `AssetTypeToParts`? `AssetPartsByType`?

Reply:
Not sure I understand you here.  I just moved `Record<AssetType, AssetParts[]>` to it's own type.  Are you just saying the name is not good?

Reply:
I'd appreciate a more descriptive name for the type.